Legion.FrontEnd/src/events/EventsCalendar/Components/Month.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Week from './Week';
+import moment from 'moment';
+
+class Month extends React.PureComponent {
+  state = {
+    month: this.props.month,
+    selected: this.props.selected
+  }
+
+  createWeeks = (month) => {
+    var weeks = [],
+        date = month.clone().startOf('month').startOf('week'),
+        // startDate = date.clone(),
+        endDate = date.clone().add(1, 'M');
+    while (date.isSameOrBefore(endDate, 'day')) {
+      weeks.push(<span className='body' key={date.date()}><Week month={date} selected={this.props.selected} /></span>)
```

Comment:
Small note, but I think I will be removing all `key` attribute references later. It produces some warnings about replicated keys, and does not seem to have any significant importance at the moment

Reply:
You need `key` whenever you render an array of React elements.  If you have duplicate `key`'s, go back and check that you're using unique values for your `key`.  The values should be something unique to that instance, so for your "week" element, you could use the year, month, and week numbers.  It looks like you might've attempted something like that, but I wonder if you're using 'date.date()' as the key for a child component too?  You'd have to look into it.